testsuite/run_sets/init_clients.yml

```diff
@@ -6,11 +6,8 @@
 
 ## Core - Initialize clients BEGIN ##
 
-# Proxy
-  # one of: proxy_register_as_minion_with_script.feature
-  #         proxy_register_as_minion_with_gui.feature
-  #         proxy_register_as_pod.feature
-- features/init_clients/proxy_register_as_minion_with_script.feature
+- features/init_clients/proxy_traditional.feature
+- features/init_clients/proxy_container.feature
```

Comment:
Will this work if we have both proxy features enabled?
I also wonder why we have these 2 times, one here in this file, and then again in `min_proxy.yml`.

Reply:
min_proxy.yml is not in use, none of our pipelines are using it, but I did not want to remove it on this PR.

The strategy is to have both features in the yaml files, so we don't need to touch the pipelines, instead, we use Cucumber tags to skip one of them, depending the environment.